sql/parser/builtins.go

```diff
@@ -399,15 +401,46 @@ var builtins = map[string][]builtin{
 		builtin{
 			types:      typeList{},
 			returnType: DummyDate,
+			fn: func(e EvalContext, args DTuple) (Datum, error) {
+				return DDate{Time: e.StmtTimestamp.Truncate(24 * time.Hour)}, nil
+			},
+		},
+	},
+
+	"statement_timestamp": {nowImpl},
+	"current_timestamp":   {nowImpl},
+	"now":                 {nowImpl},
```

Comment:
In postgres, `now()` and `current_timestamp()` are equivalent to `transaction_timestamp()`, not `statement_timestamp()`.